src/core/prompts/system-prompt/tools/list_files.ts

```diff
@@ -23,6 +24,7 @@ const generic: ClineToolSpec = {
 			instruction: "Whether to list files recursively. Use true for recursive listing, false or omit for top-level only.",
 			usage: "true or false (optional)",
 		},
+		TASK_PROGRESS_PARAMETER,
```

Comment:
I am not sure if listing files as a tool should be a part of focus chain.

Reply:
Although if search_files is a part then perhaps list_files should be a part of it too.

CC @canvrno